db/schema.rb

```diff
@@ -19,11 +19,12 @@
   create_table "active_storage_attachments", force: :cascade do |t|
     t.string "name", null: false
     t.string "record_type", null: false
-    t.bigint "record_id", null: false
+    t.bigint "legacy_record_id"
     t.bigint "blob_id", null: false
     t.datetime "created_at", null: false
+    t.uuid "record_id"
     t.index ["blob_id"], name: "index_active_storage_attachments_on_blob_id"
-    t.index ["record_type", "record_id", "name", "blob_id"], name: "index_active_storage_attachments_uniqueness", unique: true
+    t.index ["record_type", "legacy_record_id", "name", "blob_id"], name: "index_active_storage_attachments_uniqueness", unique: true
```

Comment:
I think I initially said this is probably fine to keep in the index but I thought about this more - 

We'll get in trouble with this because we require unique on legacy_record_id but now that it's `nil` and that won't be unique

Reply:
Let me know if my understanding is correct here:

legacy record id would need to be unique if we are creating new site banners and attachments, but we are fine as long as we are only working with existing banners (this would all be happening during maintenance mode). Once we run the rake task, we can get rid of the legacy id fields, **then** we merge in the PR for multi channel mode and return from maintenance mode.

Reply:
We can do it this way

```ruby 
remove_index :active_storage_attachments, name: :index_active_storage_attachments_uniqueness
```

then add the index again once the second deploy is finished and the legacy id's are removed. 

```ruby 
add_index :active_storage_attachments, [:record_type, :record_id, :name, :blob_id], name: "index_active_storage_attachments_uniqueness", unique: true
```